tui.py

```diff
@@ -1,232 +0,0 @@
-import collections
```

Comment:
In a conversation yesterday, I got the impression that it was safe to remove insofar as you had made the decision to go with QT for development instead.

Separately, it's a second frontend codebase to maintain as program specifications evolve. Just seemed like more overhead to be lugging around, but if it's actively being used and necessary, I can add it back.